silverstripe/framework/CVE-2019-12205.yaml

```
@@ -0,0 +1,14 @@
+title:     'CVE-2019-12205: Clipboard Reflected XSS'
+link:      https://www.silverstripe.org/download/security-releases/cve-2019-12205/
+cve:       CVE-2019-12205
+branches:
+    3.0.x:
+        time:     null
+        versions: ['>=3.0.0', '<3.99.99']
```

```suggestion
        versions: ['>=3.0.0', '<4']
```

As per previous comment, would this be more accurate?

---

will change to 3.9.9